ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/tests/test_sftp_outbound_transfer.py

```diff
@@ -432,17 +428,17 @@ def test_it_executes_sftp_transfer_process_if_event_is_valid(self):
         expected_s3_download_obj_key = DEFAULT_MOCK_OBJECT_KEY
         assert actual_s3_download_obj_key == expected_s3_download_obj_key
 
-        actual_sftp_open_path = mock_sftp_client.open.call_args.kwargs["filename"]
-        expected_sftp_open_path = f"{DEFAULT_MOCK_PARTNER_CONFIG.recognized_files[0].staging_folder}/{DEFAULT_MOCK_OBJECT_FILENAME}"
-        assert actual_sftp_open_path == expected_sftp_open_path
+        actual_sftp_putfo_path = mock_sftp_client.putfo.call_args.kwargs["remotepath"]
```

Comment:
Seems like this is changing functionality, what's the reason for this?

Reply:
It appears these tests weren't updated when the Lambda's throughput issues were fixed. Namely, the Lambda _used_ to call `SFTPClient.open()` when uploading files whereas now it's using `SFTPClient.putfo()`. Since `open()` was never called, the mock was never called and therefore the unit tests were failing.

This resolves the failing unit tests and ensures they're testing the appropriate calls are still being made.

Reply:
Oh this is just test code that wasn't updated from a previous change, got it.